requirements.txt

```diff
@@ -0,0 +1,15 @@
+atomicwrites==1.3.0
+attrs==19.1.0
+chardet==3.0.4
+coverage==4.5.3
+easyimap==0.6.3
+EasyProcess==0.2.5
+more-itertools==7.0.0
+pkg-resources==0.0.0
+pluggy==0.9.0
+py==1.8.0
+pytest==4.4.0
+PyVirtualDisplay==0.2.1
+selenium==3.141.0
+six==1.12.0
+urllib3==1.24.1
```

Comment:
Is this automatically processed by pip / some other package manager?

Reply:
Yes. This is the output of `pip freeze` and can be used with `pip install -r requirements.txt` to make your pip use the same package versions as mine.
This is pretty need when combined with virtual environments, as you can have different versions of the same package for different projects and keep things consistent between our machines.
I ended up doing this instead of checking my virtual environment into the git and have to say it's the far more elegant solution :P